openspace.cfg

```diff
@@ -130,6 +129,7 @@ ModuleConfigurations = {
         WebSocketInterface = "DefaultWebSocketInterface"
     },
     CefWebGui = {
+        GuiUrl = "http://localhost:4680/#/ontouch/",
```

Comment:
Apparently we have two ways to set this now... :) We will have to figure this out before merging